packages/common/src/hooks/useQueryParams/useQueryParams.ts

```diff
@@ -28,6 +30,7 @@ export const useQueryParams = <T extends DomainObject>({
   const filter = useFilterBy(initialFilterBy);
   const sort = useSortBy(initialSortBy);
   const pagination = usePagination();
+  const [storeId] = useLocalStorage('/authentication/storeid', '');
```

Comment:
concerned that this crosses the line between pure, common functions and host concerns. The `storeId` is now a parameter though, so logically it made sense to have it in here..

Reply:
I suppose the intention was more of a simplified way of using filtering, sorting & pagination for some queries. The storeID is needed in a lot of places so I see the motivation.

I can see how all this auth stuff isn't really fitting our current structure.. I wonder about another `auth`  package.. seems excessive, but I can see it being helpful